sematic/api_client.py

```diff
@@ -794,7 +816,7 @@ def request(
             fargs=[_url(endpoint)],
             fkwargs=kwargs,
             exceptions=Exception,
-            tries=API_CALLS_TRIES if retry else 1,
+            tries=1,
```

Comment:
weirdly, the integration test seems to get stuck if I simply don't use `retry_call` instead of using `tries=1`. I verified that this issue exists independent of the main change in this PR, so we can address that separately: #832